cosmo_tester/test_suites/test_manager_upgrade/manager_upgrade_base.py

```diff
@@ -187,17 +225,14 @@ def _load_private_ip_from_env(self, workdir):
     def bootstrap_manager(self):
         self.bootstrap_blueprint = self.get_bootstrap_blueprint()
         inputs_path = self.manager_cfy._get_inputs_in_temp_file(
-                self.manager_inputs, self._testMethodName)
+            self.manager_inputs, self._testMethodName)
 
         self.manager_cfy.bootstrap(self.bootstrap_blueprint,
                                    inputs_file=inputs_path)
 
         self.upgrade_manager_ip = self.manager_cfy.get_management_ip()
         self.manager_private_ip = self._load_private_ip_from_env(
-                self.cfy_workdir)
-
-        # TODO: why is this needed?
-        self.manager_cfy.use(management_ip=self.upgrade_manager_ip)
+            self.cfy_workdir)
```

Comment:
I'm not sure it was ever needed - we bootstrap using that cfy, so the manager ip/etc are already stored, after bootstrapping.

I think maybe it was only needed because i was using a broken cfy checkout when first writing it? One that didn't store the workdir settings...